subiquity/server/controllers/install.py

```diff
@@ -627,8 +632,7 @@ async def install(self, *, context):
                 fsc = self.app.controllers.Filesystem
                 for_install_path = self.model.source.get_source(fsc._info.name)
 
-            if self.app.controllers.Filesystem.reset_partition:
-                self.app.package_installer.start_installing_pkg("efibootmgr")
+            await self.install_live_packages()
```

Comment:
It's probably not really important, but I was letting the install in the live environment run in parallel with the other bits here, whereas this will wait for the installs to be done. Was that intended?

Reply:
having zfsutils-linux up front is intended, we'll need them fairly early (before rsync).
efibootmgr is safe with the later call waiting for task completion.
Updated to make this more nuanced, by allowing zfsutils-linux to be required up front and efibootmgr can run in the background.